tests/apple-pay/applepay.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, CustomerStepPreset, PaymentStepAsGuestPreset } from '../';
+import { internalOrder, order, orderPayment, validateMerchantResponse, consignments } from './ApplePayTestMockResponse';
+
+test.describe('ApplePay', () => {
+    test.skip('ApplePay in payment step', async ({ assertions, checkout, page }) => {
```

Comment:
It looks like we're skipping this test?

Reply:
Had that as a temporary measure, removed